packages/liveblocks-client/package.json

```diff
@@ -34,10 +22,12 @@
     "format": "eslint --fix src/ && prettier --write src/",
     "lint": "eslint src/",
     "test": "jest --watch --silent --verbose --config=./jest.config.js",
-    "test-ci": "jest --silent --verbose",
-    "test-e2e": "jest --silent --verbose --config=./jest.config.e2e.js"
+    "test-ci": "jest --silent --verbose --passWithNoTests"
```

Comment:
Since all tests are removed from this package for now, this is necessary to make CI as a whole pass for now. I'll remove this flag again if/when I move some of the tests back into this repo.

Reply:
Actually I misunderstood, ignore this!  (I thought it was pass even if it fails)